spec/03-plugins/09-key-auth/04-hybrid_mode_spec.lua

```diff
@@ -59,6 +59,10 @@ for _, strategy in helpers.each_strategy({"postgres"}) do
         cluster_rpc = rpc,
         cluster_rpc_sync = rpc_sync,
       }))
+
+      helpers.pwait_until(function()
+        assert.logfile("servroot2/logs/error.log").has.line("[kong.sync.v2] full sync ends", true)
```

Comment:
assert.logfile("servroot2/logs/error.log").has.line("[kong.sync.v2] full sync ends", true, 10)

Reply:
Why should we add a timeout for `line` if we already set a timeout for `pwait_until`?

Reply:
Remove pwait_until

Reply:
done.